### ISSUE TITLE
[There's a fork!] Abandoned project? Website is down

### ISSUE THREAD

**Issue by James-Livesey** [furf/jquery-ui-touch-punch#330]
Seems like jQuery UI Touch Punch is abandoned since the linked website is down. The last commit was in 2014 anyway so it's not like there's going to be much happening around here!

**Comment by James-Livesey** [comment]
Seems like it still works though after all these years (for me anyway)!

**BIG FAT DISCLAIMER:** No, I won't help you if Touch Punch goes wrong yourself. I'm just a mere observer. Maybe open an issue and hope that one day, somebody will come across this barren land and actually answer the question for you (see relevant XKCD)

<img src="https://imgs.xkcd.com/comics/wisdom_of_the_ancients.png" alt="Go look at XKCD 979" title="XKCD 979: All long help threads should have a sticky globally-editable post at the top saying 'DEAR PEOPLE FROM THE FUTURE: Here's what we've figured out so far ...'" />

_**Figure 1:** Relevant XKCD ([source](https://xkcd.com/979/))_

**Comment by James-Livesey** [comment]
I'll close this. I'm not hoping for any replies anyway. If you do know something about jQUITP which I'd find interesting, let me know.

**Comment by RWAP** [comment]
Yes it was no longer being maintained - hence I set up my own fork with most of the changes implemented!

https://github.com/RWAP/jquery-ui-touch-punch

**Comment by James-Livesey** [comment]
Oh nice! I'll take a look and give it a go in my project!

**Comment by James-Livesey** [comment]
I may as well reopen this issue so it is the first thing people look at when they come to the issues section of this repo
